marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-curator.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We still need tests for all 3 of these new roles. I think we can get away with just reading the role from the Manage API and verifying that the role inherits all of the expected roles. Each assertion should explain why the role is being inherited.

These should be core tests - can put them in the hub.security package.